### DISCUSSION TITLE
Not able to set background image and texts alignment in canter in ipxe boot menu

### DISCUSSION THREAD

**Discussion by amardeepl11** [ipxe/ipxe#1064] [Q&A]
Hi All,

Can you please help me to set an background image and set menu text in centre of the screen on ipxe boot menu.

As well as how can i set text font size too.

thanks
Amardeep kumar

**Comment by mcb30** [comment]
Please see the documentation at https://ipxe.org/cmd/console

**Reply by amardeepl11** [reply]
Hi @mcb30  thanks , can you suggest for text alignment and font size as well.

**Reply by amardeepl11** [reply]
Hi @mcb30 , I have just used below code to my bootconfig.ipxe 

#!ipxe

dhcp
console --x 1024 --y 768 --keep
console --picture console.png --keep

chain tftp://my-ipaddress/config/boot.ipxe

when i am booting , it says console : command not found

Please suggest.

**Reply by NiKiZe** [reply]
Please read the error message carefully, it will contain an url that you should also follow and read carefully. The previous linked documentation page will also contain the information you need to get this working, especially the part about buildoptions.

There is also other existing discussions that you can search for about "console" with examples.

**Reply by amardeepl11** [reply]
Hi @NiKiZe thanks for suggestion but this what i can see while booting from pxe , can you suggest if I am using console in right config.

![image](https://github.com/ipxe/ipxe/assets/139690905/ab8a8215-f1c1-486e-ad4d-fce88117b7f4)
